Calculator.py

```diff
@@ -21,4 +21,7 @@ def cm_to_m(self, nums):
     cm_to_m_unit = 100
     result = list(map(lambda x: x/cm_to_m_unit, nums))
     return result
- 
+  def m_to_cm(self, nums):
+  	m_to_cm_unit = 100
+  	result = list(map(lambda x:m_to_cm_unit*x, nums))
```

Comment:
shouldn't this be `lambda x: x / m_to_cm_unit`

Reply:
`m_to_cm_unit` is undefined in `cm_to_m()` function.

Reply:
I don't know what happened, right now your code looks fine. Can you fix the merge conflicts and update the PR.